guides/change-management.md

```diff
@@ -0,0 +1,31 @@
+# Change management
+
+New projects and features often have questions about browser support: which browsers and features need to be supported, when it is acceptable to provide alternate displays, etc.  
+
+[Browser support standards](https://github.com/cfpb/development/blob/master/guides/browser-support.md) have been established by the Browser Support Working Group. In some cases, however, there will be reasons that customers, product owners, or engineering will want to change what they are required to support in their application. This could be because a new leading-edge technology delivers value that cannot be achieved otherwise, or the customer's target market is not our normal user base and will use a known set of browsers, etc.
```

Comment:
Because we're changing from being browser support specific to ongoing FEWD topics, should we change `Browser Support Working Group` to `FEWD team`? They're one and the same, and anyone reading this outside the team presumably has a better ideas who the members of FEWD are vs any working group.

Reply:
TRIVIAL: If the lines were broken at 80 char we could comment on individual lines here.

Reply:
Updated to Front End Development team/FEWD team throughout in https://github.com/cfpb/development/pull/165/commits/ecb097c4ce1d64a4bbcedcd537d2f783932cb11d